woodwork/tests/utils/test_read_file.py

```diff
@@ -42,12 +43,15 @@ def test_read_file_validation_control(mock_validate_accessor_params, sample_df_p
 @pytest.mark.parametrize(
     "filepath, exportfn, kwargs, pandas_nullable_fix",
     [
-        ("sample.csv", ("to_csv", {"index": False}), {}, True),
-        ("sample.csv", ("to_csv", {"index": False}), {"content_type": "csv"}, True),
+        ("sample.csv", ("to_csv", {"index": False}), {"logical_types": {
+            'signup_date': Datetime()}}, True),
+        ("sample.csv", ("to_csv", {"index": False}), {"content_type": "csv", "logical_types": {
+            'signup_date': Datetime()}}, True),
```

Comment:
Why did we make the changes in this file? It's been long enough since I've looked at this, that I'm now struggling to recall the details.

Reply:
I think signup_date wasn't getting inferred as a Datetime. Revisiting it though, it doesn't seem to be a problem. I've removed it